filebeat/docs/filebeat-filtering.asciidoc

```diff
@@ -0,0 +1,39 @@
+[[filtering-data]]
+=== Filtering the Exported Data
+
+When your use case requires only a subset of the data exported by Filebeat, you can <<filebeat-filtering-overview,use Filebeat config options to filter the data>>, or you can <<generic-filtering-overview-filebeat,use the generic filtering provided by libbeat>>.
+
+[[filebeat-filtering-overview]]
+==== Filebeat Config Options for Filtering
+
+You can specify configuration options in the `filebeat` section of the config file to define regular expressions that
+match the lines you want to include and/or exclude from the output. The supported options are <<include-lines,`include_lines`>>, <<exclude-lines,`exclude_lines`>>, and <<exclude-files,`exclude_files`>>. 
+
+For example, you can use the `include_lines` option to export any lines that start with "ERR" or "WARN":
+
+[source,yaml]
+-------------------------------------------------------------------------------------
+include_lines: ["^ERR", "^WARN"]
```

Comment:
I think it would be helpful to include enough context with the example to know where `include_lines` belongs within the config file.

```
filebeat:
  prospectors:
      paths:
        - /var/log/myapp/*.log
      include_lines: ["^ERR", "^WARN"]
```